src/test/java/com/google/cloud/tools/io/FilePermissionsTest.java

```diff
@@ -79,8 +79,9 @@ public void testSubDirectoryCannotBeCreatedDueToPreexistingFile() throws IOExcep
     }
   }
   
-  @Test
+  @Test // Unix only
   public void testDirectoryCannotBeCreatedDueToUnwritableParent() throws IOException {
```

Comment:
Apparently [Windows doesn't actually support read-only directories](http://stackoverflow.com/questions/25034068/how-to-make-directory-read-only-in-windows-through-java), and `setWritable(false) == false` (i.e., operation failed).  So I think this change, and the other similar changes, would be better encoded as:
```
Path dir = Files.createDirectory(Paths.get(parent.toString(), "child"));
Assume.assumeTrue(dir.toFile().setWritable(false));
```